src/features/todos/Todo.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { useDispatch } from "react-redux";
+import { toggleTodo } from "./todosSlice";
+
+const TodoList = ({ allTodos }) => {
```

Comment:
Instead of passing props here, it would be better to use `useSelector` and just import your array of todos